bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/internal/LGThinQBindingConstants.java

```diff
@@ -0,0 +1,385 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.lgthinq.internal;
+
+import static java.util.Map.entry;
+
+import java.io.File;
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.lgthinq.lgservices.model.DeviceTypes;
+import org.openhab.core.OpenHAB;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link LGThinQBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Nemer Daud - Initial contribution
+ */
+@NonNullByDefault
+public class LGThinQBindingConstants {
```

Comment:
To make this a bit more readable and maintainable, it would be nice to arrange the constants by category and sort them alphabetically. Not preventing a merge, but yeah it would be an improvement.